utils/utils.go

```diff
@@ -138,3 +138,10 @@ func CopyDetachable(dst io.Writer, src io.Reader, keys []byte) (written int64, e
 	}
 	return written, err
 }
+
+// CopyFile copies a file from a source to destination path
+func CopyFile(sourcePath, outputPath string) error {
+	args := []string{sourcePath, outputPath}
+	_, err := ExecCmd("cp", args...)
```

Comment:
I would rather you do this with a iowriter/ioreader combination.

Reply:
done